src/features/common/Layout/UserLayout/UserLayout.tsx

```diff
@@ -169,21 +169,24 @@ function UserLayout(props: any): ReactElement {
   const [activeSubMenu, setActiveSubMenu] = React.useState('');
 
   React.useEffect(() => {
+    
     if (router) {
       for (const link of navLinks) {
+        
         if (router.router?.asPath === link.path) {
-          setactiveLink(link.path);
+            setactiveLink(link.path);
         } else if (link.subMenu && link.subMenu.length > 0) {
           const subMenuItem = link.subMenu.find((subMenuItem: any) => {
             return subMenuItem.path === router.router?.asPath;
-          });
+          })
           if (subMenuItem) {
+           if (subMenuItem && typeof subMenuItem !== 'undefined') {
```

Comment:
First condition also checks if the the property is `undefined` or not. How does the second condition works here?